pyvista/plotting/utilities/sphinx_gallery.py

```diff
@@ -66,6 +66,10 @@ class Scraper:
 
     """
 
+    def __repr__(self):  # pragma: no cover
+        """Return a stable representation of the class."""
+        return f"<{self.__class__.__name__}"
```

Comment:
Is this missing an ending `>` on purpose? And should this have an explicit `"class"` in its name? I figured this meant to look something like
```py
>>> Foo()
<__main__.Foo object at 0x7f7faf006cd0>
```
except without the "object at ..." part.

Reply:
Specifically I'd have expected
```suggestion
        return f"<{self.__class__.__name__}>"
```

Reply:
@larsoner Could you have your answer about this?
> Is this missing an ending > on purpose?

Reply:
I just forgot the `>`. Adding `object` is fine too if you want

Reply:
OK, thanks for confirming. I'll open a PR later.